jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -37,10 +37,13 @@
      ))
   (:state (&rest args)
    ":state calls with :wait-until-update by default, since Fetch publishes /joint_states from body and gripper at almost same frequency"
```

Comment:
@708yamaguchi @knorth55 could you add more documentation here?
```
(send *ri* :state :worldcoords) returns current coordinates in '/world' freame
(send *ri* :state :potentio-vector) ...
...
...
```
or update https://github.com/jsk-ros-pkg/jsk_robot/tree/master/jsk_fetch_robot#common

Reply:
@k-okada 
Thank you for your review.

@knorth55 
Could you check and merge https://github.com/knorth55/jsk_robot/pull/127 ?

Reply:
Thanks, I merged the PR.